Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ruby:2.4.3-alpine3.7
 
-ARG GOLANG_VERSION="1.9.2-r1"
+ARG GOLANG_VERSION="1.9.4-r0"
```

Comment:
Compiling Phusion Passenger in Alpine is a pain. I tried to use Go 1.12, Alpine 3.10 and newer versions of Ruby (2.6.5 & 2.7.0), but it kept breaking on the compile. Seems the issues was in Ruby-land and the interface with system level dependencies.

You can see some of the stack traces in https://github.com/Intellection/passenger-exporter/issues/10. Figured we can kick the can down the road. Possibly then move away from Alpine too to to make life easier.

Reply:
Noted.